### ISSUE TITLE
Issue in querying from emulator

### ISSUE THREAD

**Issue by MohsenMahdieh** [Azure/azure-cosmosdb-dotnet#242]
I am having issue querying from Emulator. Basically what I am doing is:

- connecting to Emulator on https://localhost:8081/
- begin able to create collection on database 
- being able to add to collection
- when I am trying to query what I have added to collection, it doesn't return any data. the code to query data is:
`var client = await this.connectionHandler.InitialiseClient().ConfigureAwait(false); // create DocumentClient
            var option = new FeedOptions() { MaxItemCount = maxItemCount, EnableCrossPartitionQuery = true };
            var query = await client.CreateDocumentQuery<TEntity>(this.connectionHandler.CollectionLink, option).Where(expression).AsDocumentQuery().ExecuteNextAsync<TEntity>().ConfigureAwait(false);
            var items = query.AsEnumerable();`

Basically the item is empty. What I should emphasize is as soon as I switch to Azure DocumentDb the same code works. 

Any idea?

**Comment by MohsenMahdieh** [comment]
Excellent support guys 👎
